test/remote/gateways/remote_opp_test.rb

```diff
@@ -18,25 +18,25 @@ def setup
       merchant_transaction_id: "active_merchant_test_complete #{time}",
       address: address,
       description: 'Store Purchase - Books',
-        #      riskWorkflow: true,
-        #      testMode: 'EXTERNAL' # or 'INTERNAL', valid only for test system
+      #      riskWorkflow: true,
+      #      testMode: 'EXTERNAL' # or 'INTERNAL', valid only for test system
 
-        billing_address: {
+      billing_address: {
```

Comment:
Here, too.

Reply:
Here `billing_address` is aligned with `description` on line 20.

Reply:
Yes, but the `}` is aligned neither with the hash key nor with the opening brace, which are the two styles honored by RuboCop. There are a couple ways to address this, but probably the best would be to unindent the entire collection of hashes so that the closing braces align with the hash keys (which looks like a three-space unindent, fi I'm reading this properly, for lines 25–30, 32–38, and 40-whenever that block ends, inclusive)